packages/mobile/App/ui/navigation/screens/patientProgramRegistration/PatientProgramRegistrationDetails.tsx

```diff
@@ -66,8 +66,7 @@ export const PatientProgramRegistrationDetails = ({ route }) => {
   const [pprCondition] = useBackendEffect(
     async ({ models }) =>
       models.PatientProgramRegistrationCondition.findForRegistryAndPatient(
-        patientProgramRegistration.programRegistryId,
-        patientProgramRegistration.patientId,
+        patientProgramRegistration.id,
       ),
     [patientProgramRegistration],
   );
```

Comment:
Do we need this `useBackendEffect` now? Or can we just go `patientProgramRegistration?.conditions`?

Reply:
yes since the conditions aren't attached